cloud: pause or resume a cluster (#10031)
pingcap/docs: tidb-cloud/pause-or-resume-tidb-cluster.md

---
title: Pause or Resume a TiDB Cluster
summary: Learn how to pause or resume a TiDB cluster.
---

# Pause or Resume a TiDB Cluster

TiDB Cloud allows you to easily pause and resume a cluster that is not in operation at all times.

Comparing with backup and restore, pausing and resuming a cluster takes less time and keeps your cluster state information (including cluster version, cluster configurations, and TiDB user accounts).

> > **Note:**
> >
> > Currently, you cannot pause a [Developer Tier cluster](/tidb-cloud/select-cluster-tier.md#developer-tier). Once a Developer Tier cluster remains idle for 24 hours, the cluster hibernates automatically, and you can resume it by [connecting to your cluster](/tidb-cloud/connect-to-tidb-cluster.md) or clicking **Resume** for the cluster in the TiDB Cloud console.

Reviewer: ```suggestion
> **Note:**
>
> You cannot pause a [Developer Tier cluster](/tidb-cloud/select-cluster-tier.md#developer-tier). See [Automatic hibernation and resuming] for Developer Tier.
```

Reviewer: How about putting this into Limitation section.

Author: This is not a limitation for Dedicated Tiers, but only for Developer Tiers. What about still keeping it as a note and adding similar info to https://docs.pingcap.com/tidbcloud/select-cluster-tier#developer-tier-special-terms-and-conditions?

> ## Prerequisites

Reviewer: Is it better to change it to "Limitations" and put it at the end of this doc?

Author: It depends on when we want users to notice this info, before the pausing, or after the pausing.

> - You cannot pause your cluster if it has any [Changefeeds](/tidb-cloud/changefeed-overview.md). You need to delete the existing Changefeeds ([Delete Sink to Apache Kafka](/tidb-cloud/changefeed-sink-to-apache-kafka.md#delete-a-sink) or [Delete Sink to MySQL](/tidb-cloud/changefeed-sink-to-mysql.md#delete-a-sink)) before pausing the cluster.

Reviewer: ```suggestion
<!--- - You cannot pause your cluster if it has any [Changefeeds](/tidb-cloud/changefeed-overview.md). You need to delete the existing Changefeeds ([Delete Sink to Apache Kafka](/tidb-cloud/changefeed-sink-to-apache-kafka.md#delete-a-sink) or [Delete Sink to MySQL](/tidb-cloud/changefeed-sink-to-mysql.md#delete-a-sink)) before pausing the cluster. --->
```

Changefeed is not public.

> - You can pause your cluster only when it is in the **Normal** state. If a cluster is in other states such as **Importing** or **Scaling**, you must wait for the current operation to be completed before pausing the cluster.

Reviewer: ```suggestion
- You can pause your cluster only when it is in the **Normal** state. If a cluster is in other states such as **Scaling**, you must wait for the current operation to be completed before pausing the cluster.
- You can not pause your cluster if the cluster is being imported, you can either wait for the import task finished or cancel the import task on TiDB Cloud UI.

```

- You cannot pause your cluster when it is backing up data, you can either wait for the current backup jobs to be completed or [delete the running backup job](/tidb-cloud/backup-and-restore.md#delete-a-running-backup-job).

> ## Pause a TiDB cluster

Reviewer: Will we add Open API method if API becomes public?

Author: If we want users to try API,  we can add a link here. It depends on when API is ready for trial.

When a cluster is paused, note the following:

- TiDB Cloud stops collecting monitoring information of the cluster, you cannot read data from or write data to the cluster, and you cannot import or back up data.

> - TiDB Cloud only charges the cluster for storage cost and does not charge for any other costs.

Reviewer: ```suggestion
- Only the following costs will be charged: 
    1. Node Storage Cost
    2. Data Backup Cost
    3.  Data Transfer – Load Balancing

```

---
title: Pause or Resume a TiDB Cluster
summary: Learn how to pause or resume a TiDB cluster.
---

# Pause or Resume a TiDB Cluster

TiDB Cloud allows you to easily pause and resume a cluster that is not in operation at all times.

The pause does not affect your data stored in the cluster but only stops the monitoring information collection and computing resource consumption. After the pause, you can resume your cluster at any time.

Comparing with backup and restore, pausing and resuming a cluster takes less time and keeps your cluster state information (including cluster version, cluster configurations, and TiDB user accounts).

> **Note:**
>

> > You cannot pause a [Developer Tier cluster](/tidb-cloud/select-cluster-tier.md#developer-tier). See [Automatic hibernation and resuming](/tidb-cloud/select-cluster-tier.md#automatic-hibernation-and-resuming) for Developer Tier.

Author: See https://github.com/pingcap/docs/pull/10031#discussion_r948635926. We will still keep the current sentences.

- You cannot pause your cluster when a data import task is going on. You can either wait for the import task to be completed or cancel the import task.
- You cannot pause your cluster when a backup job is going on. You can either wait for the current backup job to be completed or [delete the running backup job](/tidb-cloud/backup-and-restore.md#delete-a-running-backup-job).

<!--- - You cannot pause your cluster if it has any [Changefeeds](/tidb-cloud/changefeed-overview.md). You need to delete the existing Changefeeds ([Delete Sink to Apache Kafka](/tidb-cloud/changefeed-sink-to-apache-kafka.md#delete-a-sink) or [Delete Sink to MySQL](/tidb-cloud/changefeed-sink-to-mysql.md#delete-a-sink)) before pausing the cluster. --->

## Pause a TiDB cluster

When a cluster is paused, note the following:

- TiDB Cloud stops collecting monitoring information of the cluster.
- You cannot read data from or write data to the cluster.
- You cannot import or back up data.
- Only the following costs will be charged:

    - Node Storage Cost
    - Data Backup Cost

>     - Data Transfer – Load Balancing

Reviewer: ```suggestion
```
@DanielZhangQD and I have decided to remove this.
